Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -13,10 +13,15 @@ private enum SearchListSection: Int {
     case Search
 }
 
+protocol SearchViewControllerDelegate: class {
```

Comment:
I don't understand why you want a special delegate for this. Can we not just have a UrlPickerDelegate? What do you anticipate is special about the search panel compared to the normal home panels? Why is it worth this effort to keep it separate?